Gemfile

```diff
@@ -162,3 +146,30 @@ unless ENV["APPLIANCE"]
     gem "rspec-rails",      "~>3.5.0"
   end
 end
+
+#
+# Custom Gemfile modifications
+#
+
+# Load developer specific Gemfile
+#   Developers can create a file called Gemfile.dev.rb containing any gems for
+#   their local development.  This can be any gem under evaluation that other
+#   developers may not need or may not easily install, such as rails-dev-boost,
+#   any git based gem, and compiled gems like rbtrace or memprof.
+#
+#   To develop a gem locally and override its source to a checked out repo
+#   you can use this helper method in Gemfile.dev.rb e.g.
+#
+# override_gem 'manageiq-ui-classic', :path => File.expand_path("../manageiq-ui-classic", __dir__))
```

Comment:
I think this comment and method should be [above the "Load developer specific Gemfile" comment](https://github.com/ManageIQ/manageiq/pull/13418/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R153)